draft-ietf-quic-transport.md

```diff
@@ -4099,8 +4101,9 @@ bytes.  A server MAY also immediately close the connection by sending a
 CONNECTION_CLOSE frame with an error code of PROTOCOL_VIOLATION; see
 {{immediate-close-hs}}.
 
-The server MUST also limit the number of bytes it sends before validating the
-address of the client; see {{address-validation}}.
+Until the server validates the client address, it MUST limit the
```

Comment:
I don't think this change is necessary anymore.

Reply:
I agree with @martinthomson. The amplification limit disappears when the server validates the address. The padding requirement/recommendation applies to all Initial packets, because you still have to probe the MTU, and the client is often not sure if the server has validated it.

Reply:
Thank you folks. I think there's consensus that padding should better not be tied to path validation. Addressed the text in 0410b60. PTAL.